### PR TITLE
[Localization] modifier-select-ui-handler Locales

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes?
Locales for item selection menu (modifier-select-ui-handler) between battles

## Why am I doing these changes?
To localize this menu and for accessibility

## What did change?
If a translation is provided, this menu will be in the set language
Number format adapting to the language (ex : English "1,000" for a thousand and "1 000" in French for a thousand)

**Only one minor visual issue :**
The cursor position doesn't adjusts according to text length, it disappears under the text if it is longer than the original text, so if anybody has a solution to provide...
image

### Screenshots/Videos
**Before** (French)
image

**After** (French)
image
image

## How to test the changes?
Being in Classic or Endless with a Lock Capsule and at least one item held by any Pokémon of you team to unlock "Lock rarities" and "Transfer" options in the menu.

## Checklist
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?
- [X] Have I tested the changes (manually)?
    - [X] Are all unit tests still passing? (`npm run test`)
- [X] Are the changes visual?
  - [X] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by Adri1** [comment]
Updated with Korean locales

File: src/locales/ko/modifier-select-ui-handler.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const modifierSelectUiHandler: SimpleTranslationEntries = {
+  "transfer": "Transfer",
```
Comment (returntoice) [inline src/locales/ko/modifier-select-ui-handler.ts:4]:
```suggestion
  "transfer": "도구 이동",
```

File: src/locales/ko/modifier-select-ui-handler.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const modifierSelectUiHandler: SimpleTranslationEntries = {
+  "transfer": "Transfer",
+  "reroll": "Reroll",
```
Comment (returntoice) [inline src/locales/ko/modifier-select-ui-handler.ts:5]:
```suggestion
  "reroll": "새로고침",
```

File: src/locales/ko/modifier-select-ui-handler.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const modifierSelectUiHandler: SimpleTranslationEntries = {
+  "transfer": "Transfer",
+  "reroll": "Reroll",
+  "lockRarities": "Lock Rarities",
```
Comment (returntoice) [inline src/locales/ko/modifier-select-ui-handler.ts:6]:
```suggestion
  "lockRarities": "레어도 고정",
```

File: src/locales/ko/modifier-select-ui-handler.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const modifierSelectUiHandler: SimpleTranslationEntries = {
+  "transfer": "Transfer",
+  "reroll": "Reroll",
+  "lockRarities": "Lock Rarities",
+  "transferDesc": "Transfer a held item from one Pokémon to another.",
```
Comment (returntoice) [inline src/locales/ko/modifier-select-ui-handler.ts:7]:
```suggestion
  "transferDesc": "포켓몬이 지닌 도구를 다른 포켓몬에게 옮깁니다.",
```

File: src/locales/ko/modifier-select-ui-handler.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const modifierSelectUiHandler: SimpleTranslationEntries = {
+  "transfer": "Transfer",
+  "reroll": "Reroll",
+  "lockRarities": "Lock Rarities",
+  "transferDesc": "Transfer a held item from one Pokémon to another.",
+  "rerollDesc": "Spend money to reroll your item options.",
+  "lockRaritiesDesc": "Lock item rarities on reroll (affects reroll cost).",
```
Comment (returntoice) [inline src/locales/ko/modifier-select-ui-handler.ts:9]:
```suggestion
  "rerollDesc": "소지금을 사용하여 선택 가능한 도구들을 새로고침합니다.",
  "lockRaritiesDesc": "새로고침할 때 도구 레어도를 고정합니다(새로고침 비용 상승함).",
```

**Review by returntoice — CHANGES_REQUESTED** [review]
ko/modifier-select-ui-handler.ts changes suggested

**Review by returntoice — APPROVED** [review]

**Review by Tempo-anon — APPROVED** [review]

**Comment by Tempo-anon** [comment]
@Adri1 Merged this in but it caused a game crash so I reverted it. It looks to be a git merge conflict where a variable was renamed
`    this.scene.updateAndShowLuckText(750);`

**Comment by Adri1** [comment]
... Oh great...

**Comment by Adri1** [comment]
@Tempo-anon Should be okay now, I just retested it and in doesn't crash anymore on the item selection screen

**Pull request merged**